### PR TITLE
Update README.md with runners config

### PR DESCRIPTION
**Pull request by cglukas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,5 +69,26 @@ In that case, you can use the `nuke-testrunner`
 rye run nuke-testrunner -n <path_to_nuke> -t ./tests
 ```
 
+There is also the option to configure your different runners in a `runners.json`.
+This way, you don't need to specify the full nuke path all the time. 
+It's very convenient if you want to test multiple nuke versions in parallel.
```
Comment (gillesvink) [inline README.md:74]: Try to remove very, reads better without it

File: README.md [hunk 2]
```diff
@@ -69,5 +69,26 @@ In that case, you can use the `nuke-testrunner`
...
+
+The json file is structured like this:
```
Comment (gillesvink) [inline README.md:76]: Could we add besides a template maybe two examples? One simple and one with everything added?

File: README.md [hunk 3]
```diff
@@ -69,5 +69,26 @@ In that case, you can use the `nuke-testrunner`
 rye run nuke-testrunner -n <path_to_nuke> -t ./tests
 ```
```
Comment (gillesvink) [inline README.md:71]: Would be good to add a small header here, 3x hashtag I guess

**Review by gillesvink — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (cglukas) [reply README.md]: I think examples would be better places in a wiki or tutorial. Wdyt?

File: README.md (hunk 3 above)
Comment (cglukas) [reply README.md]: I moved the section. It's better suited for the how to use section.

**Review by gillesvink — APPROVED** [review]
Great!

**Pull request merged**